doc/mgr/dashboard.rst

```diff
@@ -6,33 +6,29 @@ Ceph Dashboard
 Overview
 --------
 
-The Ceph Dashboard is a built-in web-based Ceph management and monitoring
-application through which you can inspect and administer various aspects
-and resources within the cluster. It is implemented as a :ref:`ceph-manager-daemon` module.
-
-The original Ceph Dashboard that was shipped with Ceph Luminous started
-out as a simple read-only view into run-time information and performance
-data of Ceph clusters. It used a very simple architecture to achieve the
-original goal. However, there was growing demand for richer web-based
-management capabilities, to make it easier to administer Ceph for users that
-prefer a WebUI over the CLI.
-
-The new :term:`Ceph Dashboard` module adds web-based monitoring and
-administration to the Ceph Manager. The architecture and functionality of this new
-module are derived from
-and inspired by the `openATTIC Ceph management and monitoring tool
-<https://openattic.org/>`_. Development is actively driven by the
-openATTIC team at `SUSE <https://www.suse.com/>`_, with support from
-companies including `Red Hat <https://redhat.com/>`_ and members of the Ceph
-community.
-
-The dashboard module's backend code uses the CherryPy framework and implements
-a custom REST API. The WebUI implementation is based on
-Angular/TypeScript and includes both functionality from the original dashboard
-and new features originally developed for the standalone version
-of openATTIC. The Ceph Dashboard module is implemented as an
-application that provides a graphical representation of information and statistics
-through a web server hosted by ``ceph-mgr``.
+The Ceph Dashboard is a built-in web-based Ceph management-and-monitoring
+application used to inspect and administer resources in the cluster. It is
+implemented as a :ref:`ceph-manager-daemon` module.
```

Comment:
Suggest:
>The Ceph Dashboard is a web-based management and monitoring tool that can be used to view cluster and component status and administer resources.

Reply:
I like that better.

@anthonyeleven, don't merge this one until Ilya and Neha and Dan and Josh have been consulted and satisfied. I want to make sure to update the contributors to the Dashboard while respecting the historical contributions of SUSE. But I want to make sure that they are satisfied before I start ripping out vital organs.

Reply:
Ack, was already leaving the merge to you.